msu/utilities/type_checkers.nut

```diff
@@ -65,3 +65,14 @@
 		}
 	}
 }
+
+::MSU.requireAllButTypes <- function( _typeArray, ... )
```

Comment:
requireAllButTypes implies that the types should be **all** except these given types. Whereas what we actually want is that the type can be **any** except the given. 
Renaming suggestions: `requireNotTypes`, `requireAnyTypeBut`, `requireAnyTypeExcept`

Reply:
I like `requireAnyTypeExcept`

Reply:
Agreed